docs/docs/development/setup.mdx

```diff
@@ -273,50 +185,5 @@ This step pulls down quite a bit of tooling. Go grab a cup of coffee, it can tak
 :::
 
 :::info
-If you're using Docker, you're done with setup! You must restart the container at this point. The easiest way to do so is to close the VS Code window, verify that the container has stopped in Docker Dashboard, and reopen the container with VS Code.
-
 Once your container is restarted, proceed to [Building and Flashing](development/build-flash.mdx).
 :::
-
-### Export Zephyr CMake package
-
-This allows CMake to load the code needed to build ZMK.
-
-```sh
-west zephyr-export
-```
-
-### Install Zephyr Python Dependencies
-
-Some additional Python dependencies are listed in Zephyr's `scripts/requirements.txt` file.
-
-<Tabs
-defaultValue="linux"
-groupId="python-os"
-values={[
-{label: 'Linux', value: 'linux'},
-{label: 'Windows', value: 'win'},
-{label: 'macOS', value: 'mac'},
-]}>
-<TabItem value="linux">
-
-```sh
-pip3 install --user -r zephyr/scripts/requirements.txt
-```
-
-</TabItem>
-<TabItem value="win">
-
-```sh
-pip3 install -r zephyr/scripts/requirements.txt
```

Comment:
My main concern with this is related to my comment here https://github.com/zmkfirmware/zmk/pull/2216#issuecomment-2049051599 about really only needing `requirements-base.txt` deps, and our Docker images not having everything needed to install any "extras" that would get pulled in by following the upstream Zephyr docs.

Reply:
I see - valid point, I was just refactoring what is already there. I've added a note on this point, that's the best solution I can think of.

Reply:
Resolved in reshuffling.